### PR TITLE
Small improvements on the CFP Application

### PR DESCRIPTION
**Pull request by dwursteisen**

### CODE REVIEW

**Comment by aheritier** [comment]
Docker startup is ok.
I will review all others changes this evening

**Comment by dwursteisen** [comment]
I updated the MR with the last commit. 

Capture d’écran 2019-11-29 à 13 46 54

File: conf/messages [hunk 1]
```diff
@@ -141,7 +142,7 @@ preview.warn2=in the main CFP screen.
 next.action=Select next action:
 
 # Proposal type
-key.label=Keynote, 20mn, Thursday and Friday
+key.label=Keynote, 20mn, Thursday and Friday - ⛔️ Only on the invitation of the CFP Team.
```
Comment (aheritier) [inline conf/messages:145]:
```suggestion
key.label=Keynote, 20mn, Thursday and Friday - ⛔️ Only on invitation.
```

File: conf/messages.fr [hunk 2]
```diff
@@ -127,7 +128,7 @@ preview.warn2=dans le tableau de la page d''accueil.
 presubmit.reminder=Lorsque vous êtes prêt à envoyer votre proposition, n''oubliez pas d''utiliser le bouton "Soumettre"
 
 # Proposal type
-key.label=Keynote, 20mn, jeudi et vendredi matin
+key.label=Keynote, 20mn, jeudi et vendredi matin - ⛔️ Uniquement sur invitation de l'équipe du CFP
```
Comment (aheritier) [inline conf/messages.fr:131]:
```suggestion
key.label=Keynote, 20mn, jeudi et vendredi matin - ⛔️ Uniquement sur invitation.
```

File: conf/messages [hunk 3]
```diff
@@ -302,6 +304,7 @@ freepass=Yes
 no.freepass=No
 speakers.updated=Speakers''s list updated
 noOther.speaker=---- No other speaker ----
+proposal.noOtherSpeakers=Your proposal type doesn't allow other speakers.
```
Comment (aheritier) [inline conf/messages:307]:
```suggestion
proposal.noOtherSpeakers=Your proposal type doesn't allow to add another speaker.
```

File: conf/application.conf [hunk 4]
```diff
@@ -106,7 +106,7 @@ play {
 }
 
 # Set to false will close the CFP for speakers
-cfp.isOpen=${CFP_IS_OPEN}
+cfp.isOpen=true
```
Comment (aheritier) [inline conf/application.conf:109]:
```suggestion
cfp.isOpen=${CFP_IS_OPEN}
```
It is important to keep it configurable from an environment variable, it is how @nicmarti changes it in the production env

**Review by aheritier — APPROVED** [review]
Really good changes @dwursteisen 
I proposed few cosmetic changes (and 1 more important - to not hardcode the CFP OPEN flag)
@nicmarti Can you review it, probably accept my changes and merge please ?

File: conf/application.conf (hunk 4 above)
Comment (aheritier) [reply conf/application.conf:109]: @dwursteisen Note that docker is loading the file run.env and you can customize it

**Comment by aheritier** [comment]
@nicmarti  I am not sure why it's not marked as merged too here.
Just take care of my comments in this PR. The `cfp.isOpen` is the most annoying error
I can do a PR

**Pull request closed without merging**